From 563059641b737cac8970ace9557226fc56d21fc2 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 21 Dec 2023 11:48:09 +0200
Subject: [PATCH] [SW-0] uart: increase ring buffer size

during time critical sections, a ring buffer is used for prints
collection. this buffer is not large enough to store all the prints
and that might hide error information. the buffer is increased to 512
bytes

Change-Id: I2505cc51aee3070d02cbbce928c5eb3a48aa4277
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/lib/common/uart.c b/lib/common/uart.c
index 0d27aed..1672def 100644
--- a/lib/common/uart.c
+++ b/lib/common/uart.c
@@ -16,7 +16,7 @@
 #define UART_SR_RXFULL			(1 << 1) /* RX Buffer Full */
 #define STR_UINT_SZ			8
 #define BITS_IN_SINGLE_CHAR		4
-#define RING_BUFF_SIZE			SZ_256
+#define RING_BUFF_SIZE			SZ_512
 
 struct device_uart uart = {
 	.regs_initialized = false,
